typescript/src/start_script.ts

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env node
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { execSync } from 'child_process';
+import * as fs from 'fs';
+import * as path from 'path';
+import * as rimraf from 'rimraf';
+import * as readline from 'readline';
+const argv = require('yargs').argv;
+
+const cwd = process.cwd();
+
+const GOOGLE_GAX_PROTOS_DIR = path.join(
+  cwd,
```

Comment:
There is no guarantee that the `cwd` will contain `node_modules`. I would say in most cases the generator will be run from some other place. You can use `__dirname` as your starting point to locate the protos.